ledgers/xero/.travis.yml

```diff
@@ -0,0 +1,7 @@
+---
+sudo: false
+language: ruby
+cache: bundler
+rvm:
```

Comment:
Can you copy the root travis one so we test all major versions of ruby and edge but not as a blocker?